py/parse_and_notify.py

```diff
@@ -186,16 +233,27 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
         return UpdatedTopic(
             name=raw_content["topic"],
             dry_run=raw_content["dryRun"],
-            error=raw_content["error"],
             error_message=raw_content["errorMessage"],
             change_set=change_set,
         )
 
 
 def main():
-    token = os.getenv("DATADOG_API_KEY")
-    assert token is not None, "No Datadog token in DATADOG_API_KEY env var"
-    notifier = Notifier(datadog_api_key=token)
+    # API key for datadog
+    dd_token = os.getenv("DATADOG_API_KEY")
+    # Signing secret for KCP eng-pipes webhook
+    slack_secret = os.getenv("KAFKA_CONTROL_PLANE_WEBHOOK_SECRET")
```

Comment:
nit: i'm not sure that "kafka control plane" means anything in our codebases, would `TOPICCTL_WEBHOOK_SECRET` be more explicit?